src/main/webapp/app/overview/student-questions/student-questions.component.ts

```diff
@@ -122,7 +122,6 @@ export class StudentQuestionsComponent implements OnInit, AfterViewInit {
     addQuestion(): void {
         const studentQuestion = new StudentQuestion();
         studentQuestion.questionText = this.studentQuestionText;
-        studentQuestion.author = this.currentUser;
```

Comment:
would you need to remove this in the answer on the client as well?

Reply:
What should be improved is the consistency, that is correct - but in fact, no matter if `studentQuestion.author` and `studentQuestionAnswer.author` are set in the client, as they are not deserialized, they will be effectively set at the server side anyways. I suggest removing this statement in both, `student-questions.component.ts` (already removed) and `student-questions-row.component.ts` where it is currently set, do you agree?

Reply:
incorporated in d6b9cc6

Reply:
thanks

Reply:
as far as I can see `this.currentUser` is not used anymore and could be removed in this file including the assignment in line 48. I suggest to do that in a follow-up PR